CMakeLists.txt

```diff
@@ -1,4 +1,8 @@
+if(APPLE)
+cmake_minimum_required(VERSION 3.15.1 FATAL_ERROR)
```

Comment:
Leave the original line here, move this one to macosx/

```cmake
# top of CMakeLists.txt
cmake_minimum_required(VERSION 3.12 FATAL_ERROR)

# top of macosx/CMakeLists.txt
cmake_minimum_required(VERSION 3.15.1 FATAL_ERROR)
```

Reply:
wasn't possible with `project`, but made possible with `enable_language`: done in 8d80cbb1cd12ccb256b9846f65415dc914309422